provider/data_source_wireguard_config_document.go

```diff
@@ -219,6 +230,7 @@ var wgTemplate = template.Must(template.New("wg").Parse(wgTemplateStr))
 
 type WgPeerConfig struct {
 	PublicKey           string
+	Description         []string
```

Comment:
Just internally for the struct/template I might rename it `Comments` 🤔 - but dw about changing that I'll just do that in the merge commit if I do.